### DISCUSSION TITLE
Using kubectl patch to patch but be able to exclude based on service name

### DISCUSSION THREAD

**Discussion by cmwatts1974** [open-policy-agent/community#280] [Community]
I am using the following command to replace the value of NodePort to ClusterIP on a cluster and want to do it but need to exclude some services how would I edit to exclude said services?
kubectl patch svc nginx-ingress  --type='json' -p '[{"op":"replace","path":"/spec/type","value":"ClusterIP"}]'

**Comment by anderseknert** [comment]
Hi @cmwatts1974! Can you please elaborate on what role OPA plays here? This seems like it would be a better question for the Kubernetes Slack, or am I missing something? 🙂
